executor/executor.cc

```diff
@@ -1273,6 +1330,16 @@ uint64 swap(uint64 v, uint64 size, uint64 bf)
 	if (bf != binary_format_bigendian)
 		fail("bad binary format in swap: %llu", bf);
 	switch (size) {
+#if GOOS_windows
+	case 2:
+		return _byteswap_ushort(v);
```

Comment:
I would prefer if we do:
```
#define htobe16 _byteswap_ushort
...
```
at the top where we include the header.

Reply:
I made the suggested change.